src/backend/src/services/teams.services.ts

```diff
@@ -31,8 +32,11 @@ export default class TeamsService {
     if (!team) {
       throw new NotFoundException('Team', teamId);
     }
+    const activeProjects = team.projects.filter((project) => {
```

Comment:
lets just send all projects like i said and also you need a line break above this becuase these are different concepts (not that this line is going to exist anyways)